### ISSUE TITLE
Nimbus blockchain tests errors

### ISSUE THREAD

**Issue by winsvega** [status-im/nimbus-eth1#1366]
Here 
http://retesteth.ethdevops.io/results/log//2022-12-13-1670902186-nimbus.txt

Nimbus is the fastest client it seems. 
Is there a dev chat I can join to share the test run bugs?

**Comment by jangko** [comment]
wow, happy to see it producing some results.
you can join [nimbus-eth1 channel](https://discord.gg/jxYnpVb7) in our [discord server](https://discord.gg/tGGGpmgR).
(invite link expires in 7 days).

**Comment by winsvega** [comment]
All tests run is about 100/200 seconds faster then geth. 

Here the blockchain tests errors:
```
info: (ValidBlocks/bcStateTests/BLOCKHASH_Bounds_Berlin, fork: Berlin, block: 1)
info: (ValidBlocks/bcStateTests/SuicidesMixingCoinbase_Berlin, fork: Berlin, block: 2)
info: (ValidBlocks/bcStateTests/suicideCoinbaseState_Berlin, fork: Berlin, block: 1)
info: (ValidBlocks/bcStateTests/testOpcode_40_Berlin, fork: Berlin, block: 1)
```

**Comment by winsvega** [comment]
```
 (ValidBlocks/bcStateTests/SuicidesMixingCoinbase_Berlin, fork: Berlin, block: 2)
Error: The command '/home/ubuntu/.retesteth/nimbus/start.sh --state.fork Berlin --state.reward 2000000000000000000 --state.chainid 1 --input.alloc /dev/shm/2d7c1acc-aece-4d53-a4a8-748523cfc468/alloc.json --input.txs /dev/shm/2d7c1acc-aece-4d53-a4a8-748523cfc468/txs.rlp --input.env /dev/shm/2d7c1acc-aece-4d53-a4a8-748523cfc468/env.json --output.basedir /dev/shm/2d7c1acc-aece-4d53-a4a8-748523cfc468 --output.result out.json --output.alloc outAlloc.json' exited with 1024 code. 

(ValidBlocks/bcStateTests/testOpcode_40_Berlin, fork: Berlin, block: 1)
Error: Importing raw RLP block, block was expected to be valid! (if it was intended, check that it is not in Valid blocks test suite) Error importing raw rlp block: The command '/home/ubuntu/.retesteth/nimbus/start.sh --state.fork Berlin --state.reward 2000000000000000000 --state.chainid 1 --input.alloc /dev/shm/2d7c1acc-aece-4d53-a4a8-748523cfc468/alloc.json --input.txs /dev/shm/2d7c1acc-aece-4d53-a4a8-748523cfc468/txs.rlp --input.env /dev/shm/2d7c1acc-aece-4d53-a4a8-748523cfc468/env.json --output.basedir /dev/shm/2d7c1acc-aece-4d53-a4a8-748523cfc468 --output.result out.json --output.alloc outAlloc.json' exited with 1024 code. (ValidBlocks/bcStateTests/testOpcode_40_Berlin, fork: Berlin, block: 1)

```

nimbus t8n tool fails to process the input data

**Comment by jangko** [comment]
Hai @winsvega, I've merged PR #1381 to fix the above issue.

**Comment by winsvega** [comment]
Blockchain tests pass
